### ISSUE TITLE
Missing btoa when I'm trying to use this on react-native

### ISSUE THREAD

**Issue by jeloagnasin** [xaviergonz/mobx-keystone#63]
I'm trying to use this on our monorepo which includes a react-native repo because I'm tired of trying to make mobx-state-tree to work with TS 😂 but this library assumes that there's a `btoa` or `Buffer` defined in the global variable.

I've setup a [simple repo](https://github.com/jeloagnasin/test-keystone) where the issue exists.

In App.tsx in the repo above, importing `createRootStore` doesn't work because it will immediately initialize the [`localBaseId`](https://github.com/xaviergonz/mobx-keystone/blob/master/packages/lib/src/globalConfig/globalConfig.ts#L38) which requires the `btoa` or `Buffer`. So I have to polyfill it first before importing.

This doesn't work:

```tsx
import { createRootStore } from "./src/store"
import { TodoListView } from './src/TodoListView';

global.Buffer = global.Buffer || require('buffer').Buffer;
```

This works:

```tsx
import { TodoListView } from './src/TodoListView';

global.Buffer = global.Buffer || require('buffer').Buffer;
const { createRootStore } = require('./src/store');
```

**Comment by terrysahaidak** [comment]
I can confirm this. UUID works totally fine with react native, but it seems like it's bundling/using node.js version of UUID with react native for some reason.

**Comment by terrysahaidak** [comment]
I can confirm this. `btoa` API doesn't exist in React Native (I should have noticed it in the PR). We can polyfill it or replace with some alternative for non-browsers.

**Comment by xaviergonz** [comment]
should be fixed in 0.26.5. it will fallback to a btoa ponyfill only if Buffer / btoa is not present (so compatibility is not lost).

**Comment by jeloagnasin** [comment]
@xaviergonz Cool! I will take a look at it and close this issue as soon as I've verified it. Thank you so much for the quick response!
